source/managing-teams/what-engineering-managers-do.html.md

```diff
@@ -0,0 +1,94 @@
+---
+title: What Engineering Managers do
+meta_description: >
+  Engineering managers are the point of contact for clients at Nebulab, as well as looking after
+  the growth and well-being of their team.
+---
+
+At Nebulab, we strongly believe in the value of competent, supportive management.
+
+When the company was 10 people, the Directors could look after each project personally, interacting
+with clients and mentoring the team. While it was a lot of fun, that kind of structure doesn't
+really scale, so we set out to find a better alternative and created the role of the Engineering
+Manager (EM).
+
+Engineering Managers are typically responsible for the performance of their teams, doing all that's
+needed to ensure that their team performs at full capacity, and for ensuring everyone on the team is
+happy and proud to be a part of Nebulab.
+
+At Nebulab, EMs report directly to the Directors, and their top two priorities are: 
+
+1. **Overseeing the execution of client projects end-to-end.** While we provide some guidance and
+   best practices, EMs have complete freedom in how they structure and run their projects. In
+   addition to coordinating engineering work and acting as informal project managers, you may find
+   them helping out with product management, hiring, and anything else that's needed to ensure the
+   client's success.
+2. **Looking after the well-being and growth of their team.** This might mean a lot of different
+   things depending on the project and team composition. At the very least, it involves holding
+   regular 1:1s with all team members, protecting their Investment Time and work-life balance,
+   looking after their professional progression through our Growth Framework, and assisting the
```

Comment:
What about linking `Growth Framework` to the `Personal Growth` chapter?

Reply:
Done!